tests/test_kernel_magics.py

```diff
@@ -53,19 +54,42 @@ def test_start_session():
     line = ""
     assert not magic.session_started
 
-    magic._do_not_call_start_session(line)
+    ret = magic._do_not_call_start_session(line)
```

Comment:
Usually I prefer to have a meaningful variable name even if it's in unit test, this will help any new person to this code section to get the intent of the test easily.